e2e/testcases/login/5-validate-langiage-drop-down.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { test, expect, type Page } from '@playwright/test'
+import { LOGIN_URL } from '../../constants'
+import { ensureLoginPageReady } from '../../helpers'
+
+let page: Page
```

Comment:
Don't you get Page as a parameter to different test cases `, async ({ page }) => {` ?